### DISCUSSION TITLE
Backup/export settings

### DISCUSSION THREAD

**Discussion by Wannabe-01** [Tichau/FileConverter#291] [Ideas]
Hi

I have setup my file converter as i like it, however, when i get a new pc, i always has to start anew. is there some ways, i can backup my settings and restore on a new pc witoyt having to do all the settings manuel. i have looked in to appdata, and regedit. but even the few keys i find in there, is no good on a new pc.

**Comment by glecas** [comment]
I belive you can backup and later restore this file:
`%localappdata%\FileConverter\Settings.user.xml`

**Comment by Tichau** [comment]
Exactly, we've talked about this here: https://github.com/Tichau/FileConverter/issues/381
